packages/core/src/sync/sync-api.ts

```diff
@@ -242,14 +265,17 @@ export class SyncApi implements ISyncApi {
   async _handleBlockProofs({ block, reorganized }: BlockProofsListenerEvent): Promise<void> {
     this.currentBlock = block.number
 
+    // reorg
     if (reorganized) {
       await this._addSyncJob(HISTORY_SYNC_JOB, {
+        jobType: SyncJobType.Reorg,
         fromBlock: block.number - BLOCK_CONFIRMATIONS,
         toBlock: block.number,
         models: Array.from(this.modelsToSync),
       })
     } else {
       await this._addSyncJob(CONTINUOUS_SYNC_JOB, {
+        jobType: SyncJobType.Reorg,
```

Comment:
Not important but can we change this to "new block". This is the case when a new block is heard